templates/web/borsetshire/around/postcode_form.html

```diff
@@ -1,36 +0,0 @@
-[% UNLESS c.user_exists %]
-<div class="homepage-login-hint">
-    <div class="container">
-        <h2>Psssst… Want to see behind the curtain?</h2>
-        <p>Try out FixMyStreet as a customer service rep, a highways inspector, or a site administrator. <a href="/auth">Sign in</a> to begin!</p>
-    </div>
-</div>
-[% END %]
-
-<div id="front-main">
-    <div id="front-main-container">
-        [% INCLUDE 'around/intro.html' %]
-
-        [%
-            question = c.cobrand.enter_postcode_text || loc('Enter a nearby street name and area');
-        %]
-
-        <form action="[% c.uri_for('/around') %]" method="get" name="postcodeForm" id="postcodeForm">
-            <label for="pc">[% question %]:</label>
-            <p class="form-hint" id="pc-hint">[% tprintf(loc('e.g. ‘%s’ or ‘%s’'), c.cobrand.example_places) %]</p>
-            <div>
-                <input type="hidden" name="pc" value="[% c.cobrand.problems.first.postcode | html %]">
-                <input type="text" name="_pc" value="[% pc | html %]" id="pc" size="10" maxlength="200" aria-describedby="pc-hint" required>
```

Comment:
This functionality of a hidden pc and _pc for what they enter (so it always takes them to the pins) has been lost, could we have it back?